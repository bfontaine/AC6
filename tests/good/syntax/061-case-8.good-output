Line 1, characters 31-32: Error (Parsing)
  Unknown error.

