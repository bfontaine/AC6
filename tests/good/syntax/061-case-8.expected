val x =
  case { { First -> x, Tag } => x }
